scripts/src/bus.lua

```diff
@@ -1128,6 +1128,8 @@ if (BUSES["ISA"]~=null) then
 		MAME_DIR .. "src/devices/bus/isa/myb3k_com.h",
 		MAME_DIR .. "src/devices/bus/isa/myb3k_fdc.cpp",
 		MAME_DIR .. "src/devices/bus/isa/myb3k_fdc.h",
+		MAME_DIR .. "src/devices/bus/isa/eis_sad8852.cpp",
+		MAME_DIR .. "src/devices/bus/isa/eis_sad8852.h",
```

Comment:
Can you please keep source files in order within these blocks?  It makes it far easier to keep track of stuff.

Reply:
Sure, do you want me to sort all of the ISA cards? It looked very unsorted to me so I couldn't find a good spot to add new entries